README.md

```diff
@@ -43,6 +43,7 @@ const saml = new SAML(options);
 - `audience`: expected saml response Audience (if not provided, Audience won't be verified)
 - `cert`: the IDP's public signing certificate used to validate the signatures of the incoming SAML Responses, see [Security and signatures](#security-and-signatures)
 - `privateKey`: see [Security and signatures](#security-and-signatures).
+- `signingCert`: the service provider's public signing certificate used to embed in AuthnRequest in order for the IDP to validate the signatures of the incoming SAML Request, see [Security and signatures](#security-and-signatures)
```

Comment:
Would this be the same signing cert as used in `generateServiceProviderMetadata`? It appears that way, in which case, would it seem reasonable to modify `generateServiceProviderMetadata()` to take this value if the `signingCert` argument isn't provided?

Reply:
Yes, this is same certificate used in generateServiceProviderMetadata(). So, I given same parameter name for this.

Reply:
I've merged `master` into your branch because it has some changes to `generateServiceProviderMetadata` and I wanted to make sure that your code fit with them.